manifests/policy/operators/grc-policy-propagator.yaml

```diff
@@ -62,35 +62,8 @@ spec:
             weight: 35
       containers:
       - args:
-        - --secure-listen-address=0.0.0.0:8443
-        - --upstream=http://127.0.0.1:8383/
-        - --logtostderr=true
-        - --v=6
-        - --tls-min-version=VersionTLS12
-        image: quay.io/stolostron/kube-rbac-proxy:2.12.0-SNAPSHOT-2024-09-18-21-46-30
-        imagePullPolicy: IfNotPresent
-        name: kube-rbac-proxy
-        ports:
-        - containerPort: 8443
-          name: https
-          protocol: TCP
-        resources: {}
-        securityContext:
-          allowPrivilegeEscalation: false
-          capabilities:
-            drop:
-            - ALL
-          privileged: false
-          readOnlyRootFilesystem: true
-          runAsNonRoot: true
-        terminationMessagePath: /dev/termination-log
-        terminationMessagePolicy: File
-      - args:
-        - --compliance-history-api-host=0.0.0.0
         - --health-probe-bind-address=:8081
-        - --metrics-bind-address=:8383
-        - --leader-elect
-        - --enable-webhooks=false
+        - --metrics-bind-address=0.0.0.0:8443
```

Comment:
It's not going to be HTTPS based on the volume mounts I see so perhaps keeping it as 8383 makes sense since 8443 implies HTTPS.

Reply:
changed to 8383